libraryAPI/pkg/service/book_test.go

```diff
@@ -37,6 +40,10 @@ func TestCreateBooks(t *testing.T) {
 
 func TestGetBooks(t *testing.T) {
 	t.Run("Successfully got list of books", func(t *testing.T) {
+		configPath := "./../../configs/config.env"
+		config, err := config.New(configPath)
```

Comment:
you don't need to call `config.New` here, you can just create your own config from Config struct here, in this case, you won't need to read any file (redundant dependency)